### DISCUSSION TITLE
Would GPU help?

### DISCUSSION THREAD

**Discussion by ceewanna** [Breakthrough/DVR-Scan#111] [General]
I am a new user. My setup is as follows.
1. I had a cron job running every minute on each wifi ip camera of which the rtsp stream is copied into mp4 file (-codec copy, no extra encoding or decoding is done).
3. In the cron, after the 1-minute mp4 is done. DVR-Scan is activated to produce motion mp4. The option -m ffmpeg is used here where ffmpeg-output-args = -crf 32 -codec copy. The original 1-minute video is then moved to archive.
4. The cameras are 2M x 3 and 4M x 3.

The PC machine is AMD without any external GPU right now. I have been monitoring the CPU load. It hits the roof top most of the time. My observation is that the ffmpeg stream copy requires only a few percentage of CPU load. I wonder if getting a GPU would help in my case.

**Comment by Breakthrough** [comment]
On my computer, running in CUDA mode results in ~17% CPU load, whereas CPU mode tries to use as much CPU as possible to process as quick as it can.  However I cannot say for certain what the results will be if you use a GPU, as the program is not very optimized right now.

If processing time is less of a concern, you can limit the CPU usage of the DVR-Scan process to avoid excess CPU load, or set it's priority/nicencess lower so it does not interrupt other tasks. Hope this helps!
